main/libinput/APKBUILD

```diff
@@ -10,7 +10,7 @@ license="MIT"
 makedepends="eudev-dev libevdev-dev meson mtdev-dev"
 subpackages="$pkgname-dev $pkgname-doc $pkgname-libs"
 options="!check"
-source="https://freedesktop.org/software/$pkgname/$pkgname-$pkgver.tar.xz"
+source="https://freedesktop.org/software/libinput/$pkgname-$pkgver.tar.xz"
```

Comment:
Any particular reason why this was changed?

Reply:
I think this change is good. It makes it easier to copy/paste the source url to a browser to look for new release. The use of `$pkgname` in there brings little value if any.